data/meetings/d2-sober-sunrise.yaml

```diff
@@ -8,11 +8,10 @@ time:
   Thursday: ["06:45"]
   Friday: ["06:45"]
 address: zoom,Online-Only,,
-note: Meeting ID:  944-241-0663    Password:  Rise@645
+note: | 
+     Meeting ID:  944-241-0663    Password:  Rise@645
```

Comment:
Don't think we even need this part, since it'd included below, but just in case it wasn't added by mistake, I left it here